models/junos/security_policy/junos_security_policies.yml

```diff
@@ -0,0 +1,514 @@
+---
+GENERATOR_VERSION: '1.0'
```

Comment:
The template used is relatively an older template, all the parent options are not required only the part under documentation makes sense.
Ref : https://github.com/ansible-network/resource_module_models/blob/master/models/ios/ntp_global/ios_ntp_global.yaml